packages/material-ui/src/Step/Step.test.js

```diff
@@ -93,4 +94,50 @@ describe('<Step />', () => {
       assert.strictEqual(wrapper.find('.hello-world').length, 1);
     });
   });
+
+  describe('step connector', () => {
+    it('should not exist when non-alternativeLabel Step', () => {
+      const wrapper = shallow(
+        <Step
+          label="Step One"
+          alternativeLabel={false}
+          orientation="horizontal"
+          connector={<StepConnector />}
+        />,
+      );
+      assert.strictEqual(wrapper.children().length, 0);
+    });
+
+    it('should exist when alternativeLabel Step', () => {
+      const wrapper = shallow(
+        <Step
+          label="Step One"
+          alternativeLabel
+          orientation="horizontal"
+          connector={<StepConnector />}
+        />,
+      );
+      assert.strictEqual(wrapper.children().length, 1);
+    });
+
+    it('should pass active prop to connector when next step is active', () => {
+      const wrapper = shallow(
+        <Step
+          label="Step One"
+          alternativeLabel
+          orientation="horizontal"
+          connector={<StepConnector />}
+          nextStepState={{ active: true }}
+        />,
+      );
+
+      assert.strictEqual(
+        wrapper
+          .children()
```

Comment:
That's testing implementation details. Using `find(StepConnector)` would be more robust.

Reply:
You're right. I feel that I initially tried that but it didn't work. Trying again I see it does work so I have made the appropriate changes.